Dockerfile

```diff
@@ -727,8 +729,10 @@ COPY --link --exclude=usr/bin/ctr --from=pkg-containerd-amd64 / /rootfs
 COPY --link --from=pkg-dosfstools-amd64 / /rootfs
 COPY --link --from=pkg-e2fsprogs-amd64 / /rootfs
 COPY --link --exclude=usr/share --from=pkg-systemd-udevd-amd64 / /rootfs
+COPY --link --exclude=usr/share --from=pkg-systemd-udevd-amd64 /usr/share/spdx/systemd.spdx.json /rootfs/usr/share/spdx/systemd.spdx.json
```

Comment:
would this work an `exclude` of `usr/share` and then copying from there itself, I think we don't need the exclude directive here